src/main/java/uk/gov/hmcts/probate/transformer/solicitorexecutors/LegalStatementExecutorTransformer.java

```diff
@@ -49,6 +49,8 @@ public void createLegalStatementExecutorLists(List<CollectionMember<AdditionalEx
                                              List<CollectionMember<AdditionalExecutorNotApplying>> execsNotApplying,
                                              CaseData caseData) {
 
+
+
```

Comment:
Empty lines, not sure if you meant to add these.

Reply:
I'll remove - cheers!